apps/site/lib/site_web/controllers/schedule/line/helpers.ex

```diff
@@ -111,6 +116,41 @@ defmodule SiteWeb.ScheduleController.Line.Helpers do
     RouteStops.by_direction(stops[route.id], shapes, route, direction_id)
   end
 
+  @spec get_shuttle_paragraphs(Plug.Conn.t()) :: [Phoneix.HTML.safe()]
+  def get_shuttle_paragraphs(conn) do
+    conn
+    |> shuttle_paragraphs_by_line()
+    |> Enum.map(&get_paragraph/1)
+    |> Enum.map(&post_process_paragraph/1)
+    |> Enum.map(&render_paragraph(&1, conn))
+  end
+
+  @spec shuttle_paragraphs_by_line(Plug.Conn.t()) :: [binary()]
+  defp shuttle_paragraphs_by_line(conn) do
+    common_shuttle_paragraphs = [
+      "paragraphs/content-list/shuttles-sidebar-project",
+      "paragraphs/custom-html/shuttles-boilerplate"
+    ]
+
+    line_shuttle_paragraphs =
+      case conn.assigns.route.id do
+        "Red" -> ["paragraphs/custom-html/shuttles-sidebar-red-line"]
+        "Orange" -> ["paragraphs/custom-html/shuttles-sidebar-orange-line"]
+        "Green" -> ["paragraphs/custom-html/shuttles-sidebar-green-line"]
+      end
```

Comment:
@amaisano we haven't talked about the Blue Line so far, but let's add a default clause to this `case`  that returns an empty list, just to cover ourselves.